scripts/prepare_doc_publication.py

```diff
@@ -60,6 +60,7 @@ def parse_arguments():
     'getting_started_python.md': 'Python',
     'cmake_options_and_variables.md': 'CMake Options and Variables',
     'op_coverage.md': 'XLA HLO Operation Coverage',
+    'e2e_coverage.md': 'Tensorflow End to End Coverage',
```

Comment:
Looking at the screenshot in the PR description, I'd like this to fit on a single line with the current CSS on standard screen sizes. Maybe abbreviate to `TF End to End Coverage` or `TensorFlow E2E Coverage` to match the permalink?

Reply:
Done.